tests/framework/ROM/TimeSeries/ARMA/MultipleFourier/correlated_0.csv

```diff
@@ -0,0 +1,101 @@
+53,-0.996398308961,-2.36755924709,1.31918106898,3.37461124333
+54,-1.19974640346,-2.89687258125,1.06953892639,3.10258520099
+55,-1.30901699437,-3.2360679775,0.587785252292,2.35114100917
+56,-1.31918106898,-3.37461124333,0.0967618536362,1.56261791913
+57,-1.23479628594,-3.32115115501,-0.180543273519,1.17993993851
+58,-1.06953892639,-3.10258520099,-0.106728590793,1.47519866942
+59,-0.844843789354,-2.76134116867,0.317041800174,2.44373770528
+60,-0.587785252292,-2.35114100917,0.951056516295,3.80422606518
+61,-0.328406995374,-1.93166197024,1.57007250302,5.1047195075
+62,-0.0967618536362,-1.56261791913,1.94908324472,5.8942199463
+63,0.0800483669535,-1.29784052094,1.94908324472,5.8942199463
+64,0.180543273519,-1.17993993851,1.57007250302,5.1047195075
+65,0.190983005625,-1.2360679775,0.951056516295,3.80422606518
+66,0.106728590793,-1.47519866942,0.317041800174,2.44373770528
+67,-0.0672896856689,-1.88719273143,-0.106728590793,1.47519866942
+68,-0.317041800174,-2.44373770528,-0.180543273519,1.17993993851
+69,-0.620759491513,-3.1010719548,0.0967618536362,1.56261791913
+70,-0.951056516295,-3.80422606518,0.587785252292,2.35114100917
+71,-1.2776001555,-4.49236663326,1.06953892639,3.10258520099
+72,-1.57007250302,-5.1047195075,1.31918106898,3.37461124333
+73,-1.80113169569,-5.58649279401,1.19974640346,2.89687258125
+74,-1.94908324472,-5.8942199463,0.713118485857,1.67690343884
+75,-2.0,-6.0,5.75707791727e-15,1.2248943914e-14
+76,-1.94908324472,-5.8942199463,-0.713118485857,-1.67690343884
+77,-1.80113169569,-5.58649279401,-1.19974640346,-2.89687258125
+78,-1.57007250302,-5.1047195075,-1.31918106898,-3.37461124333
+79,-1.2776001555,-4.49236663326,-1.06953892639,-3.10258520099
+80,-0.951056516295,-3.80422606518,-0.587785252292,-2.35114100917
+81,-0.620759491513,-3.1010719548,-0.0967618536362,-1.56261791913
+82,-0.317041800174,-2.44373770528,0.180543273519,-1.17993993851
+83,-0.0672896856689,-1.88719273143,0.106728590793,-1.47519866942
+84,0.106728590793,-1.47519866942,-0.317041800174,-2.44373770528
+85,0.190983005625,-1.2360679775,-0.951056516295,-3.80422606518
+86,0.180543273519,-1.17993993851,-1.57007250302,-5.1047195075
+87,0.0800483669535,-1.29784052094,-1.94908324472,-5.8942199463
+88,-0.0967618536362,-1.56261791913,-1.94908324472,-5.8942199463
+89,-0.328406995374,-1.93166197024,-1.57007250302,-5.1047195075
+90,-0.587785252292,-2.35114100917,-0.951056516295,-3.80422606518
+91,-0.844843789354,-2.76134116867,-0.317041800174,-2.44373770528
+92,-1.06953892639,-3.10258520099,0.106728590793,-1.47519866942
+93,-1.23479628594,-3.32115115501,0.180543273519,-1.17993993851
+94,-1.31918106898,-3.37461124333,-0.0967618536362,-1.56261791913
+95,-1.30901699437,-3.2360679775,-0.587785252292,-2.35114100917
+96,-1.19974640346,-2.89687258125,-1.06953892639,-3.10258520099
+97,-0.996398308961,-2.36755924709,-1.31918106898,-3.37461124333
+98,-0.713118485857,-1.67690343884,-1.19974640346,-2.89687258125
+99,-0.371807513904,-0.869196066867,-0.713118485857,-1.67690343884
```

Comment:
Could you reduce the lines of outputs of the tests? if possible

Reply:
No, in fact, many more lines should be included to assure more accurate statistics; this is a balance between accuracy and time to run.